Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,76 @@
+class Tarantool < Formula
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-558-ga70c9ba-src.tar.gz"
+  sha256 "a07cb6cc92bb7f05cc35f738bbde627b655df62ac7dbcd272716581c7b6bf5b1"
+
+  devel do
+    url "https://github.com/tarantool/tarantool.git", :branch => "1.6", :shallow => false
+    version "9999"
+  end
+
+  head do
+    url "https://github.com/tarantool/tarantool.git", :branch => "master", :shallow => false
+    version "1.6.*"
+  end
+
+  option "with-debug", "Build Debug version"
+  option "with-test", "Run Tests after building"
+
+  depends_on "cmake"    => :build
+  depends_on "readline" => :build
+
+  if build.with? "test"
+    depends_on "python-daemon" => [:python, "daemon", :build]
+    depends_on "pyyaml"        => [:python, "yaml",   :build]
+  end
+
+  def install
+    args = []
+    if build.with? "debug"
+      ENV.enable_warnings
+      ENV.deparallelize
+      args << "-DCMAKE_BUILD_TYPE=Debug"
+      ohai "Building with Debug"
+    else
+      args << "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+      ohai "Building with Release"
+    end
+    args << "-DCMAKE_INSTALL_SYSCONFDIR=#{prefix}/etc"
+    args << "-DCMAKE_INSTALL_LOCALSTATEDIR=#{prefix}/var"
+    args << "-DREADLINE_ROOT=/usr/local/Cellar"
```

Comment:
`READLINE_ROOT` isn't needed anymore, it was hack for 10.6(?? i don't remember exactly). I'll delete it.